docs: say that syz-verifier is currently broken

**Describe the bug**
I I'm trying to build syz_verifier, but when executing `make verifier runner executor`according to the commands in the documentation, I find that syz_verifier is not generated in /bin. Is there any other additional action required?
**To Reproduce**
```
make verifier runner executor
Makefile:31: run command via tools/syz-env for best compatibility, see:
Makefile:32: https://github.com/google/syzkaller/blob/master/docs/contributing.md#using-syz-env
go list -f '{{.Stale}}' ./sys/syz-sysgen | grep -q false || go install ./sys/syz-sysgen
make: *** No rule to make target 'runner'.  Stop.
make: *** Waiting for unfinished jobs....
make .descriptions
Makefile:31: run command via tools/syz-env for best compatibility, see:
Makefile:32: https://github.com/google/syzkaller/blob/master/docs/contributing.md#using-syz-env
bin/syz-sysgen
touch .descriptions
ls ./bin
syz-sysgen*
```

**Expected behavior**
generated ./bin/syz_verifier
**Additional context**
Add any other context about the problem here.

--- a/docs/syz_verifier.md
+++ b/docs/syz_verifier.md
@@ -1,3 +1,6 @@
+**`syz-verifier` is currently broken and cannot be compiled/used, see
+[the backlog issue](https://github.com/google/syzkaller/issues/5976).**
+
 # syz-verifier
 
 Many bugs are easy to detect: they might cause assertions failures, crash our

